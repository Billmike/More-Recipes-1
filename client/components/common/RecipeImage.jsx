import React from 'react';

const RecipeImage = () => (
  <img
    src="../public/images/image01.jpg"
    alt="Recipe name"
  />
);

export default RecipeImage;
