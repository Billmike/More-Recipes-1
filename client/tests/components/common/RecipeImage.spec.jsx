import React from 'react';

import RecipeImage from '../../../components/common/RecipeImage';

describe('Recipe Image component', () => {
  const props = {
    title: 'Rice',
    recipeImage: 'recipe image URL',
  };
  it('should render the component', () => {
    const wrapper = shallow(<RecipeImage {...props} />);

    expect(wrapper).toBeDefined();
    expect(wrapper.contains(<img src="recipe image URL" alt="Rice" />))
      .toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
