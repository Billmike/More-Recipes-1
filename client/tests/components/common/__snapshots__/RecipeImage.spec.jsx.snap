// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipe Image component should render the component 1`] = `
<img
  alt="Rice"
  src="recipe image URL"
/>
`;
